### PR TITLE
Fixed payment slate docs

### PR DESCRIPTION
**Pull request by reshmabidikar**

### CODE REVIEW

File: source/includes/_payment.md [hunk 1]
```diff
@@ -2840,15 +3275,17 @@ no content
 
 **Request Body**
 
-A JSON string representing the custom field object or objects to be added.
+A list of [Custom Field](https://killbill.github.io/slate/?java#custom-field-custom-field-resource) objects. Each object should specify at least the the `name` and `value` attribute. For example:
```
Comment (pierre) [inline source/includes/_payment.md:3278]: `the the` (and below)

**Review by pierre — APPROVED** [review]
Thanks! Just a few typos.

File: source/includes/_payment.md (hunk 1 above)
Comment (reshmabidikar) [reply source/includes/_payment.md:3278]: Fixed

File: source/includes/_invoice.md [hunk 2]
```diff
@@ -3347,11 +3347,10 @@ no content
 **Requst Body**
 
 
-A list of [Custom Field](https://killbill.github.io/slate/?java#custom-field-custom-field-resource) objects. Each object should specify at least the the `customFieldId` and `value` attribute. For example:
+A list of [Custom Field](https://killbill.github.io/slate/?java#custom-field-custom-field-resource) objects representing the fields to substitute for existing ones. Each object should specify at least the the `customFieldId` and `value` attribute. For example:
```
[the remark this answers is not in the document]
Comment (reshmabidikar) [reply source/includes/_invoice.md:3350]: Fixed

File: source/includes/_invoice.md [hunk 3]
```diff
@@ -3347,11 +3347,10 @@ no content
 **Requst Body**
```
[the remark this answers is not in the document]
Comment (reshmabidikar) [reply source/includes/_invoice.md:3347]: Fixed

**Pull request merged**